publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/expandresources/PersistedDocumentConsumptionAttributes.java

```diff
@@ -40,6 +41,9 @@ public static PersistedDocumentConsumptionAttributes createAttributes(ExpandedDa
             return new PersistedDocumentConsumptionAttributes(DOI_REQUESTS_INDEX, expandedEntry.retrieveIdentifier());
         } else if (expandedEntry instanceof ExpandedMessage) {
             return new PersistedDocumentConsumptionAttributes(MESSAGES_INDEX, expandedEntry.retrieveIdentifier());
+        } else if (expandedEntry instanceof ExpandedResourceConversation) {
+            //TODO: use different index?
+            return new PersistedDocumentConsumptionAttributes(MESSAGES_INDEX, expandedEntry.retrieveIdentifier());
```

Comment:
I do not think we need to save both the messages and the conversations. Just the conversations will do.